experiment_scheduler/task_manager/task_manager_server.py

```diff
@@ -175,14 +176,40 @@ def get_tasks_using_resource(self):
         return list(self._resource_rental_history.keys())
 
 
+class Task:
```

Comment:
It is recommended to create another file for each class.
task_manager/task.py will be appropriate

Reply:
I agree. Actually, I have a plan to combine this class with `ProcessUtill` class for refactoring. I'll move this class to another file in that PR. Let's keep it in this PR.